src/js/i18n/en-US.yml

```diff
@@ -603,8 +603,8 @@ careOptsFrontend:
         sortDroplist:
           headingText: 'Sort Options'
         sortDueOptions:
-          asc: 'Due: Oldest - Newest'
-          desc: 'Due: Newest - Oldest'
+          asc: 'Due: Sooner - Later'
+          desc: 'Due: Later - Sooner'
```

Comment:
`Due: Sooner - Later`:

![Screen Shot 2022-12-15 at 4 40 36 PM](https://user-images.githubusercontent.com/35355575/207983375-521c860e-02da-4582-a100-f0e187ce7f3e.png)

`Due: Later - Sooner`:

![Screen Shot 2022-12-15 at 4 40 49 PM](https://user-images.githubusercontent.com/35355575/207983381-39290b2b-04f9-4f55-aae2-a71d59e4e503.png)

Reply:
It's odd to me that the "up/down" icon is the same in both directions.

Reply:
We use the same icon for all options in that droplist.

Reply:
Right, I didn't mean specific to this sort. Just in general it's weird to me that we don't switch between two icons, "up" and "down".

cc @markott